src/category_theory/sites/sheaf_of_types.lean

```diff
@@ -687,6 +687,10 @@ begin
     exact PK (pullback_arrows f R) (K.pullbacks f R hR) }
 end
 
+/-- Any presheaf is a sheaf for the bottom (trivial) grothendieck topology. -/
+lemma is_sheaf_bot : is_sheaf (⊥ : grothendieck_topology C) P :=
```

Comment:
Should we do this for a `tivial_grothendieck_topology` predicate/class? Just like in topology. Because I can imagine we'll sometimes have topologies that are trivial, but not defeq to `\bot`.

Reply:
Maybe, but is there a problem with
```
example {J : grothendieck_topology C} (hJ : J = ⊥) : is_sheaf J P :=
begin
  rw hJ,
  apply is_sheaf_bot,
end
```
In particular, the notion of "the same" topology is genuine equality, not just unique isomorphism like for limits.

Reply:
I guess it works fine because `grothendieck_topology` is not a class. Good point.